.pre-commit-config.yaml

```diff
@@ -3,6 +3,7 @@ repos:
     rev: 24.3.0
     hooks:
       - id: black-jupyter
+        log_file: devtools/pre-commit.log
```

Comment:
this log is only created if black fails

Reply:
if it fails on the first pass, the run again does it delete the old log file? or does it just overwrite it with a blank one?

Reply:
I think the last one deletes it at the end of each run.